wondrous-app/components/Common/Task/TaskCard.tsx

```diff
@@ -126,25 +127,29 @@ export default function TaskCard({
   const [getTaskSubmissionsForTask] = useLazyQuery(GET_TASK_SUBMISSIONS_FOR_TASK, {
     onCompleted: (data) => {
       const taskSubmissions = data?.getTaskSubmissionsForTask;
+      let approvedTaskSubmission = null;
```

Comment:
just adding warning if there's no approved submission attached to a completed task with reward